src/styles/card.scss

```diff
@@ -0,0 +1,56 @@
+@import 'variables';
+
+.card {
+  width: $card-width;
+  padding: 20px;
+  border: 1px solid #ddd;
+  border-radius: 8px;
+  font-family: $font-family;
+  text-align: center;
+
+  &__image {
+    max-width: 100%;
+    border-radius: 4px;
+  }
+
+  &__title {
+    font-weight: 700;
+    font-size: 1.1em;
+    margin: 10px 0;
+  }
+
+  &__code,
+  &__reviews,
+  &__price {
+    font-weight: 400;
+    font-size: 0.9em;
+    color: #555;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 1.2em;
+    color: $primary-color;
+  }
+
+  &__buy {
+    display: block;
+    width: 100%;
+    padding: 10px 0;
+    color: $primary-color;
+    border: 2px solid $primary-color;
+    border-radius: 4px;
+    font-weight: 700;
+    text-align: center;
+    text-decoration: none;
+    margin-top: 10px;
+    transition:
+      background-color 0.3s,
+      color 0.3s;
+  }
+
+  &__buy:hover {
+    background-color: $primary-color;
+    color: #fff;
+  }
+}
```

Comment:
The hover effect to increase the card size by 20% is missing. According to the task requirements, you should implement a hover effect that scales the card size by 20% when hovered over. Consider using CSS transform properties to achieve this effect.